bosun.md

```diff
@@ -0,0 +1,51 @@
+# Inclusive Naming Initiative Biweekly Bosun Guide  
+
+This doc is a guide to help the delegated INI member bosun the public meeting. 
```

Comment:
```suggestion
This doc is a guide to help the delegated INI member perform the role of /bosun/ for the public meeting. 
```

It's a noun and not a typical usage, so we'll want to define and explain following this.